cmd/access/node_builder/access_node_builder.go

```diff
@@ -442,6 +446,7 @@ func (builder *FlowAccessNodeBuilder) extraFlags() {
 		flags.UintVar(&builder.rpcConf.MaxHeightRange, "rpc-max-height-range", defaultConfig.rpcConf.MaxHeightRange, "maximum size for height range requests")
 		flags.StringSliceVar(&builder.rpcConf.PreferredExecutionNodeIDs, "preferred-execution-node-ids", defaultConfig.rpcConf.PreferredExecutionNodeIDs, "comma separated list of execution nodes ids to choose from when making an upstream call e.g. b4a4dbdcd443d...,fb386a6a... etc.")
 		flags.StringSliceVar(&builder.rpcConf.FixedExecutionNodeIDs, "fixed-execution-node-ids", defaultConfig.rpcConf.FixedExecutionNodeIDs, "comma separated list of execution nodes ids to choose from when making an upstream call if no matching preferred execution id is found e.g. b4a4dbdcd443d...,fb386a6a... etc.")
+		flags.Uint64Var(&builder.followerConfig.SkipNewProposalsThreshold, "follower-skip-proposals-threshold", defaultConfig.followerConfig.SkipNewProposalsThreshold, "threshold at which new proposals are discarded rather than cached, if their height is this much above local finalized height")
```

Comment:
Instead of adding this new flag for all node types, could we just add it as a shared flag in scaffold.go?

Reply:
I didn't do this because we're dealing with 3 different engines across the different node roles, but we could use the same flag name for `compliance` on `consensus` nodes and `follower` on the others.